flow/project.py

```diff
@@ -301,13 +428,13 @@ def __init__(self, id, name, job, cmd, directives=None):
         self.directives._keys_set_by_user = keys_set_by_user
 
     def __str__(self):
-        return "{}({})".format(self.name, self.job)
+        return "{}({})".format(self.name, " ".join(map(str, self.jobs)))
```

Comment:
This is fine for a first pass, but we may want to concatenate this in the case that there are many jobs.